daemon/logger/fluentd/fluentd_test.go

```diff
@@ -22,3 +23,140 @@ func TestValidateLogOptReconnectInterval(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateLogOptAddress(t *testing.T) {
+
+	// paths to try
+	paths := []string{"/", "/some-path"}
+
+	tests := []struct {
+		addr        string
+		ports       map[string]int // combinations of addr + port -> expected port
```

Comment:
~This doesn't appear to be used as an input for any of the test cases.~

Edit: This starts to be used in "daemon/logger/fluentd: fix missing host, remove urlutil.IsTransportURL()".  Maybe remove it from this first commit and add it in the commit where it's being used?

Reply:
I've been going back and forth a bit on that one, trying to make the flow of commits "focused" and not to cause too much unrelated churn. This one was a bit tricky due to the loop for running the tests (agreed though that in the first commit it's a bit weird to already add it)

Reply:
updated, and moved this change to the commit that started using it 👍